packages/server/src/controllers/EmailsController.js

```diff
@@ -0,0 +1,46 @@
+import { User } from '../models/objection'
+import emailClient from '@aragonone/court-backend-shared/helpers/email-client'
+import sleep from '@aragonone/court-backend-shared/helpers/sleep'
+
+export default {
+  async send(req, res) {
+    const { TemplateModel, testEmail, testEmailSent } = req.body
+    let users = []
+    if (testEmailSent) {
+      users = await User.findWithoutDisabledNotifications()
+    }
+    else if (testEmail) {
+      users = [{
+        email: { email: testEmail },
```

Comment:
What do you think about having simply a recipient list? if there is any we send to those, otherwise we send to jurors

Reply:
I did it first, but then I thought maybe sending to all jurors by default is not a good idea, we should be more explicit about it.
Changed variable name to `notifyAllJurors`